### PR TITLE
Frame ui

### PR DESCRIPTION
**Pull request by MDinler**
## Description

Added all ui buttons for both live and rewound stream

## Important files to review

videoframes.py and frame.ui

### CODE REVIEW

**Comment by MDinler** [comment]
fixes #101

File: videoframes.py [hunk 1]
```diff
@@ -135,12 +132,10 @@ def __init__(self, vlc_instance, stream_url, stream_options, quality):
 
     def setup_ui(self):
         super(LiveVideoFrame, self).setup_ui()
+        # Hide forward and backward rewind button
+        # self.findChild(QtCore.QObject, "forward_rewind_button").hide()
+        # self.findChild(QtCore.QObject, "backward_rewind_button").hide()
```
Comment (botronic) [inline videoframes.py]: If these are not needed, remove them.

File: videoframes.py [hunk 2]
```diff
@@ -186,6 +181,28 @@ def rewind(self):
...
             self.rewinded.show()
 
+    def resizeEvent(self, event):
+        rect = self.geometry()
+        deleteButton_rect = self.delete_button.geometry()
+        pauseButton_rect = self.pause_button.geometry()
+        volumeSlider_rect = self.volume_slider.geometry()
+        # button_width_position = self.rewind_button.size().width()
+        # position_width = rect.width() / 2 - rewindButton_rect.width() / 2
+        # position_height = rect.height() - self.rewind_button.size().height()
```
Comment (botronic) [inline videoframes.py]: Same here.

File: videoframes.py [hunk 3]
```diff
@@ -186,6 +181,28 @@ def rewind(self):
...
+        )
+        self.pause_button.move(
+            0,
+            rect.height() - pauseButton_rect.size().height()
+        )
+        self.volume_slider.move(
+            rect.width() - volumeSlider_rect.size().width(),
+            rect.height() - volumeSlider_rect.size().height()
+        )
+# twitch.tv/esl_csgo
```
Comment (botronic) [inline videoframes.py]: And here.

**Comment by MDinler** [comment]
Fixes #109 #127 #106

**Comment by mictab** [comment]
Gonna test this now.

**Pull request merged**